requirements.txt

```diff
@@ -13,10 +13,13 @@ pycparser==2.21
 pyelftools==0.28
 Pygments==2.12.0
 pyparsing==3.0.9
-pytest==7.1.2
+pytest==7.1.2; python_version >= '3.7'
+pytest==7.0.1; python_version < '3.7'
```

Comment:
We should move pytest to dev-requirements too

Reply:
Moved it in https://github.com/pwndbg/pwndbg/commit/697c32e84bb8a36dd10ab32936dcec8011d47994 after merging this PR :)

Reply:
Ah, I was in the process of fixing this, and I saw `isort` should probably be moved too. Not sure about `iniconfig` and `packaging`, they seem like dev related tools but I'm not sure.